### ISSUE TITLE
Message buttons missing from top of screen after deleting message

### ISSUE THREAD

**Issue by jikamens** [thunderbird/thunderbird-android#6346]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App version

6.307

### Where did you get the app from?

Google Play

### Android version

Android 13

### Device model

Pixel 6

### Steps to reproduce

Sometimes after I delete a message by tapping the delete button, when it displays the next message the buttons that are supposed to be above the message, for example the delete button I just tapped, disappear.

### Expected behavior

Buttons remain visible after deleting a message and the next is displayed

### Actual behavior

Buttons disappear

### Logs

Logs are only a few seconds long and don't show the time when this last happened, don't know why. When it happens again I'll try again to capture logs.

**Comment by cketti** [comment]
A log probably won't contain any useful information in this case. But it would help if you could record a video showing the described behavior.

**Comment by jikamens** [comment]
I can't really capture a video since I never know when it's going to happen, but here's a screenshot of a message view with the buttons at the top missing which I was able to capture when it happened.

![Screenshot_20221006-100624_2](https://user-images.githubusercontent.com/1598067/194343980-6df71aaf-8862-428d-b561-c452cb3f9212.png)

Background: I had a thread of four messages in my inbox. I opened the thread and then opened the oldest message in the thread. I read the oldest three messages in the thread and deleted each of them after I was done reading it by tapping the trash can icon at the top of the message. After deleting the third one, this message was displayed, without the icons at the top.
